target/linux/ath79/generic/base-files/lib/upgrade/openmesh.sh

```diff
@@ -68,6 +68,7 @@ platform_do_upgrade_openmesh()
 	# boot anymore to Linux until it was reflashed with ap51-flash.
 	local next_boot_part="1"
 	case "$(board_name)" in
+	openmesh,a40|\
```

Comment:
So, all currently known devices have the same value?  I'd even consider to hard-code it then, or follow the function argument idea from the om2p comment. Whatever gets us rid of the case in openmesh.sh

Reply:
consider this done (but I will push it after I've changed it in all PRs)